kitsune/wiki/tests/test_views.py

```diff
@@ -684,3 +685,170 @@ def test_it_works(self):
         res = self.client.get(url)
         eq_(res.status_code, 200)
         self.assertTemplateUsed(res, 'wiki/mobile/document.html')
+
+
+class FallbackSystem(TestCase):
+    """Check per-article fallback locales are working correctly."""
+
+    def setUp(self):
+        super(FallbackSystem, self).setUp()
+        product(save=True)
+
+    def create_documents(self, locale):
+        """Create a English Document and a translated document of the locale"""
+        en = settings.WIKI_DEFAULT_LANGUAGE
+        en_content = 'This article is in English'
+        trans_content = 'This article is translated into %slocale' % locale
+        # Create an English Article and a translation in the locale
+        en_doc = document(locale=en, save=True)
+        revision(document=en_doc, content=en_content, is_approved=True,
+                 is_ready_for_localization=True, save=True)
+        trans_doc = document(parent=en_doc, title='Translated to', locale=locale, save=True)
+        # Create a revision of the translated document
+        trans_rev = revision(document=trans_doc, content=trans_content,
+                             is_approved=True, save=True)
+        # Make the revision as current revision of the translated document
+        trans_doc.current_revision = trans_rev
+        trans_doc.save()
+        # return the english document and translated document
+        return en_doc, trans_doc
+
+    def get_data_from_translated_document(self, header, create_doc_locale, req_doc_locale):
+        """Create documents and get provided locale document data"""
+        # Create a client with provided ACCEPT_LANGUAGE header information
+        client = Client(HTTP_ACCEPT_LANGUAGE=header)
+        # Create English and translated Document based on the create_doc_locale
```

Comment:
Create an English and a localized version of the document, based on create_doc_locale